### ISSUE TITLE
Beta Releases Advertised in Homebridge Logs on Reboot

### ISSUE THREAD

**Issue by mitch7391** [kiwi-cam/homebridge-broadlink-rm#379]
**Describe the bug**
Hey @kiwi-cam not sure if this is a bug or intentional, but when you have beta releases they get advertised in the homebridge logs as if it was to tell you about a new stable release.

```
{
  name: 'v4.4.7-beta.3',
  tag: { name: 'v4.4.7-beta.3' },
  isPrerelease: true,
  publishedAt: '2021-09-01T02:39:03Z',
  url: 'https://github.com/kiwi-cam/homebridge-broadlink-rm/releases/tag/v4.4.7-beta.3'
}
[UPDATE AVAILABLE] Version v4.4.7-beta.3 of Homebridge Broadlink RM Pro is available. The release notes can be found here: https://github.com/kiwi-cam/homebridge-broadlink-rm/releases/
```

**To Reproduce**
Restart homebridge.

**Expected behavior**
I think we should only see this if a new stable release is available. But maybe not even then, as most people would be using config-ui-x these days and that tells you when an update is available.

**Screenshots**
```
{
  name: 'v4.4.7-beta.3',
  tag: { name: 'v4.4.7-beta.3' },
  isPrerelease: true,
  publishedAt: '2021-09-01T02:39:03Z',
  url: 'https://github.com/kiwi-cam/homebridge-broadlink-rm/releases/tag/v4.4.7-beta.3'
}
[UPDATE AVAILABLE] Version v4.4.7-beta.3 of Homebridge Broadlink RM Pro is available. The release notes can be found here: https://github.com/kiwi-cam/homebridge-broadlink-rm/releases/
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: Windows build 10.0.19043

**Smartphone (please complete the following information):**
 - Device: iPhone 12 Pro
 - OS: 14.7.1
 - Browser: Safari
 - Version: Latest stable

**Comment by mitch7391** [comment]
Bumping this one to keep it alive.

**Comment by mitch7391** [comment]
Bumping again

**Comment by mitch7391** [comment]
Bump

**Comment by mitch7391** [comment]
Bump

**Comment by kiwi-cam** [comment]
This should now be resolved in v4.4.11-beta.1. Could you test and confirm?

**Comment by mitch7391** [comment]
Hey @kiwi-cam, just tried on `v4.4.11-beta.4` and looks to be resolved :) thanks for getting to this one!
